src/Entity/Task.php

```diff
@@ -438,10 +448,10 @@ class Task implements TaggableInterface, OrganizationAwareInterface, PackagesAwa
     private $weight;
 
     /**
-     * @var string|null
-     * @Groups({"task"})
-     */
-    private $failureReason;
+    * @var bool
+    * @Groups({"task"})
+    */
+    private $incidented = false;
```

Comment:
sorry to say that but I think it is a poor name, as it doesn't mean anything, it should be `hasIncident` or `hasIncidents`